src/components/AppConfig/LLMConfig.tsx

```diff
@@ -89,59 +89,105 @@ export function LLMConfig({
 
   return (
     <FieldSet label="OpenAI Settings" className={s.sidePadding}>
+      {' '}
```

Comment:
What's this for?

Reply:
bah, nothing. My editor really loves preserving whitespace like that